crates/bevy_core_pipeline/src/skybox/prepass.rs

```diff
@@ -0,0 +1,149 @@
+//! Skybox prepass.
```

Comment:
I'd flesh out this comment to mention that the purpose of this prepass is to write out motion vectors.

Reply:
Ah, good catch. I wrote up some docs, but that was in the previous attempt I discarded.